OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -20,9 +20,15 @@ public class WithInfantryBodyInfo : ITraitInfo, IQuantizeBodyOrientationInfo, IR
 	{
 		public readonly int MinIdleWaitTicks = 30;
 		public readonly int MaxIdleWaitTicks = 110;
-		public readonly string MoveSequence = "run";
-		public readonly string AttackSequence = "shoot";
+
+		[SequenceReference] public readonly string MoveSequence = "run";
+
+		[SequenceReference] public readonly string AttackSequence;
```

Comment:
The engine default values are not going to go away.
Nothing presented here is actor or mod specific so I'm not sure what you're requesting.